### DISCUSSION TITLE
Dropzone: Cannot scroll child components or click links

### DISCUSSION THREAD

**Discussion by nicolassanmar** [mantinedev/mantine#2991] [Q&A] [answered]
I'm trying to get a scroll area inside a dropzone like this:
```
<Dropzone>
  <ScrollArea>
    {...}
  <ScrollArea/>
</Dropzone>
```
No matter what props I pass to the Dropzone, such as `activateOnClick={false}` or `autoFocus={false}`, the `<ScrollArea>` remains fixed at the top, not reacting to scroll or keyboard presses.

I noticed a behaviour that may be related, if you have a child `<Anchor />` or any clickable component, the `<Dropzone/>`  captures the click, even on `activateOnClick={false}` , preventing the click on links to go through.

I've also tried setting the children with `style={{zIndex: 99999}}`, but it does not work.

Is there any way to prevent the Dropzone from capturing scroll and click events?

**Comment by rtivital** [comment, marked as answer]
Dropzone disables pointer events on its children for dragging events to work. You can enable all events with:
`<Dropzone styles={{ inner: { pointerEvents: 'all' } }} />`

**Reply by nicolassanmar** [reply]
Thanks! Enabling this did not prevent the dragging event from working, and let me scroll through the `<ScrollArea>`. 
@rtivital Has this been proved to cause issues? 
If not, this could be made the default.
(I tested this on the newest versions of Firefox and Chrome)

**Reply by nicolassanmar** [reply]
Making links work is more tricky, since you have to also disable the Dropzone's `activateOnClick`. 
But I think it makes lots of sense that way.
